app/src/main/res/layout/activity_transaction.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
why are we preferring `RelativeLayout` over `LinearLayout`? (refer to existing `layouts`)